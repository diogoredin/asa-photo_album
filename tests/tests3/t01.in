10 16
2 4
1 6
1 2
1 5
3 5
9 3
5 2
6 7
8 4
10 5
10 9
6 2
9 4
8 1
10 4
7 10
