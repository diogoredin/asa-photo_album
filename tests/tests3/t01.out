8 1 6 7 10 9 3 5 2 4
